python/lsst/meas/algorithms/cr.cc

```diff
@@ -35,14 +35,14 @@ namespace algorithms {
 
 namespace {
 template <typename PixelT>
-void declareFindCosmicRays(py::module& mod) {
+static void declareFindCosmicRays(py::module& mod) {
```

Comment:
Should have a newline between namespace open and the template declaration.  Would like another one after the function, and ideally a comment on the namespace close.